### DISCUSSION TITLE
qiita-cliでnpx qiita newする際のオプションで自作テンプレートの指定を可能にしてほしい

### DISCUSSION THREAD

**Discussion by KOH6** [increments/qiita-discussions#542] [要望・アイデア・不具合]
[qiita-cli](https://github.com/increments/qiita-cli)リリースいただきありがとうございます。
早速使用しておりとても便利で感動しております。

現状`npx qiita new`する際には固定のテンプレートでひな形が作成されますが、自作のテンプレートをオプションで設定できるようにしていただきたいです。
自作テンプレートをオプションで指定可能にできれば、記事のジャンルごとにTag等をあらかじめ設定したテンプレートを適用できるので、記事作成もより効率化できると考えております。

ご検討よろしくお願いいたします。

**Comment by getty104** [comment]
@KOH6 qiita-cliを使っていただきありがとうございます！
Qiitaとしてもより記事執筆しやすい体験を提供していきたいと考えておりますので、使い心地についてフィードバックをいただけて大変ありがたいです 🙏 
時期は保証できないですが検討させていただきます！

**Reply by KOH6** [reply]
承知いたしました！
はい！ぜひご検討いただけると嬉しいです！
